sharepoint_utils.py

```diff
@@ -3,3 +3,12 @@
 def print_and_log(logger, level, message):
     print(message)
     getattr(logger, level.lower())(message)
+
+def extract(content):
+    
+    parsed = parser.from_buffer(content)
```

Comment:
It looks like this could raise `TikaException` - we should rescue this exception since we don't want the whole indexing to fail because we couldn't extract the text for a single document